index.js

```diff
@@ -13,11 +13,24 @@
  @Test - Write a console.log that shows the value of `increaseLevel(10)`
 */
 
+function increaseLevel(valuePassed) {
+    valuePassed++
+    return valuePassed
+} 
+console.log(increaseLevel(10))
 /* 
  @Challenge 02 - Write a function named `makeEchoes` that loops for the number of times indicated and console.log's "Echo!" in each loop
  @Example - Sending the function a value of 5 will result in 5 "Echo!"'s in the terminal
  @Test - Make a function call of  `makeEchoes(5)`
 */
+function makeEchoes (x) {
```

Comment:
Good work. Consider avoiding single letter variables in favor or more meaningful names.